openassessment/xblock/test/test_openassessment.py

```diff
@@ -100,8 +100,8 @@ def test_student_view_workflow_error(self, xblock):
         # Expect that the page renders even if the update fails
         self.assertIn("OpenAssessmentBlock", xblock_fragment.body_html())
 
-    @ddt.data(('utc', 'April 1, 2014 00:00 UTC'),
-              ('America/Los_Angeles', 'March 31, 2014 17:00 PDT'))
+    @ddt.data(('utc', '2014-03-31 20:00'),
```

Comment:
The template-rendered dates now are contained within data elements in the html tags. No user-visible date format changes have occurred (indeed, I did a PR against edx-ui-toolkit to add a format constant to assure this would be the case).
This tests only for the data elements.